src/regions.py

```diff
@@ -216,6 +203,62 @@ def get_regions_from_name_list(region_names):
     rr = RegionRequest(request_dict)
     return rr.submit()
     
+def get_string_filter(filters, cls, key, constructed_filter):
```

Comment:
My only comment is that when i looked through the code i encountered these functions. I feel like i dont understand their purpose well enough (i didn't spend any time looking at how they are used etc.). If i was writing this code, i would add a short description for each function. I know that steve believed in self documenting code. What is your preference? Should we put a little blurb in front of functions like these or it is not necessary?

Reply:
This is a good point, I would have liked more comments as I've worked through the code as it was not self explanatory to me. I have added in a few blurbs for these functions. In the future, we may want to establish a more consistent documenting process for the code base and add in some function descriptions for most of the functions.